config/config.go

```diff
@@ -87,10 +99,51 @@ func NewConfig() (*Config, error) {
 		},
 	}
 
-	_ = cleanenv.ReadConfig("./config/config.yml", ConsoleConfig)
-	// its ok to ignore the error here, as we have default values set if the config file is not found
+	// Define a command line flag for the config path
+	configPathFlag := flag.String("config", "", "path to config file")
+	flag.Parse()
+
+	// Determine the config path
+	var configPath string
+	if *configPathFlag != "" {
+		configPath = *configPathFlag
+	} else {
+		ex, err := os.Executable()
+		if err != nil {
+			panic(err)
+		}
+		exPath := filepath.Dir(ex)
+		configPath = exPath + "/config/config.yml"
```

Comment:
change the path structure so it will be compatible across different OS --- something like filepath.join()